packages/frontend/src/components/ProjectAccess/index.tsx

```diff
@@ -20,7 +20,8 @@ const ProjectUserAccess: FC<ProjectUserAccessProps> = ({ projectUuid }) => {
     const [showProjectGroupAccessAdd, setShowProjectGroupAccessAdd] =
         useState(false);
 
-    const groupManagementEnabled = useFeatureFlagEnabled('group-management');
+    const groupManagementEnabled =
+        useFeatureFlagEnabled('group-management') || true;
```

Comment:
Won't this 'true' turn this on everywhere?

Reply:
good catch, I was doing that because feature flags act weird 🤦

Reply:
Yea...I wonder whats up there.